examples/mobile/UIComponents/components/thumbnail/index.html

```diff
@@ -45,6 +45,11 @@ <h1>
 						Thumbnail grid + text 2 lines
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="grid-reorder.html">
+						Thumbnail grid reorder
```

Comment:
Thumbnail Grid Reorder.
It is better to match the capital letters.

Reply:
Done.